api.planx.uk/tests/mockJWT.ts

```diff
@@ -1,5 +1,5 @@
 import type { Role } from "@opensystemslab/planx-core/types";
-import { sign } from "jsonwebtoken";
+import jwt from "jsonwebtoken";
```

Comment:
Required change due to this issue - https://github.com/auth0/node-jsonwebtoken/issues/655